cache/cache.cpp

```diff
@@ -281,10 +603,10 @@ int dnet_cmd_cache_lookup(struct dnet_backend_io *backend, struct dnet_net_state
 		return -ENOTSUP;
 	}
 
-	cache_manager *cache = (cache_manager *)backend->cache;
+	auto cache = reinterpret_cast<cache_manager *>(backend->cache);
```

Comment:
It is better to use `static_cast` because it gives stronger guarantees on conversions from and to `void*`.